content/docs/atom-guide/glossary.md

```diff
@@ -56,7 +56,7 @@ Draw packet
 Dynamic branching  
 : Dynamic branching is when conditional statements must be evaluated at runtime. This can cause negative performance impacts. For example, the code statement `if(sampledColor.a > 0.5)` is a dynamic branch and a common source of shader performance issues.
 
-*<!--Related to: [Design of Shader Variant Options](dev-guide/shaders/azsl/shader-variant-options/)DRAFT TOPIC--><!--, [Shader Variant Options and the Fallback Key](dev-guide/shaders/azsl/fallback-key/)DRAFT TOPIC-->*
+*<!--Related to: [Design of Shader variant options](dev-guide/shaders/azsl/shader-variant-options/)DRAFT TOPIC--><!--, [Shader variant options and the Fallback Key](dev-guide/shaders/azsl/fallback-key/)DRAFT TOPIC-->*
```

Comment:
**You can choose to do this now in this PR, or create an [Broken Link issue](https://github.com/o3de/o3de.org/issues/new?assignees=&labels=bug%2C+needs-triage%2C+website%2Fbroken-link&template=broken-link.md&title=%5BBROKEN+LINK%5D) to resolve this another time.** 

There are many links in the glossary that you can now uncomment, since we have the docs for them. If you choose to fix these links, do the following: 
* Update the link's text to the correct name of the doc. For example, change "Design of Shader variant options" to "Shader Variant Options", which is the link title of `docs\atom-guide\dev-guide\shaders\azsl\shader-variant-options.md`. 
* Capitalize "Shader Variant Options" if it is the title of the page. 
* Fix the link. The link must have the following:
  * Begins with `/docs/...`
  * Uses forward-slash `/`
  * If the link is to a page, remove `.md` at the end and replace with `/`. 
  * Please refer to the [Style Guide on Links](https://o3de.org/docs/contributing/to-docs/style-guide/format/#links).

Reply:
Fixed where applicable.